RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
 Unreleased
 ---
 
+## 1.73.1
+---
+* [*] [Space block] Fix crash when changing the height value using the text input [https://github.com/WordPress/gutenberg/pull/40053]
```

Comment:
I just noticed a small typo here ("Space" should be "Spacer"). I'll address this as part of future commits, to avoid slowing down the release process.